inbm-lib/inbm_lib/package_info.py

```diff
@@ -68,7 +68,19 @@ def check_package_status(package_name: str) -> str:
         "dpkg-query -W -f='${Status}\n' " + package_name)
     if err:
         logger.error(f"Error in getting the package's status: {package_name} Error: {err}")
-        return PACKAGE_FAIL
+        # Some packages are architecture independent.
+        # For example, using dpkg, the package is showing <pacakge>:all instead of <package>:amd64
+        # We will check the status again with independent architecture - "all"
+        try:
+            split_pkg_and_arch = package_name.split(':')
+            arch_independent_pkg = split_pkg_and_arch[0] + ':all'
+            out, err, code = shell.run(
+                "dpkg-query -W -f='${Status}\n' " + arch_independent_pkg)
+            if err:
+                return PACKAGE_FAIL
```

Comment:
let's have a new state UNKNOWN when we can't find the package

Reply:
but i'll call this a nice to have, so i will accept this PR as is

Reply:
Updated. Use PACKAGE_UNKNOWN instead of PACKAGE_FAIL when checking package.